jbpm-benchmarks/jbpm-performance-tests/src/main/java/org/jbpm/test/performance/scenario/PrepareEngine.java

```diff
@@ -9,14 +9,15 @@
 import org.kie.api.task.TaskService;
 import org.kie.api.task.model.Task;
 
+
 public class PrepareEngine {
 
     public static List<Long> createNewTasks(boolean start, int count, TaskService taskService, String runtimeManagerIdentifier) {
         String str = "(with (new Task()) { priority = 55, taskData = (with( new TaskData()) { deploymentId = '" + runtimeManagerIdentifier + "' } ), ";
         str += "peopleAssignments = (with ( new PeopleAssignments() ) { potentialOwners = [new User('" + UserStorage.PerfUser.getUserId()
                 + "')], businessAdministrators = [ new User('Administrator') ], }),";
         str += "names = [ new I18NText( 'en-UK', 'perf-sample-task')] })";
-        List<Long> taskIds = new ArrayList<Long>();
+        List<Long> taskIds = new ArrayList<>();
         while (count > 0) {
             Task task = TaskFactory.evalTask(new StringReader(str));
             long taskId = taskService.addTask(task, null);
```

Comment:
This file doesn't need to be changed at all.

Reply:
This file is still shown as changed but as it is in the other module I wouldn't change it in this PR.